pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -392,10 +397,16 @@ async def build_sync(runtime: Runtime, version: str, assembly: str, publish: boo
         exclude_arches=exclude_arches,
         skip_multiarch_payload=skip_multiarch_payload
     )
-
+    span = trace.get_current_span()
+    span.set_attributes({
+        "pyartcd.param.dry_run": runtime.dry_run,
+        "pyartcd.param.version": version,
+        "pyartcd.param.assembly": assembly,
+    })
     try:
         await pipeline.run()
         await pipeline.handle_success()
```

Comment:
If this is going to replace Redis failure counting, we should also instrument `handle_failure()` to start tracking payload issues

Reply:
Not at this moment. I would like to try that after when we have visualization and monitoring on OSD. Maybe a meter will be used for that.